pkg/quota/image/usage.go

```diff
@@ -230,38 +212,26 @@ func (c *GenericImageStreamUsageComputer) processImageStreamSpecImages(
 
 		img, exists := c.imageCache[ref.ID]
 		if !exists {
-			if c.fetchImagesFromImageStream || len(ref.ID) == 0 {
-				if len(ref.ID) > 0 {
-					img, err = c.getImageStreamImage(ref.Namespace, is.Name, ref.ID)
-					if err != nil {
-						if !kerrors.IsNotFound(err) {
-							utilruntime.HandleError(fmt.Errorf("failed to get image stream image %s/%s@%s: %v", ref.Namespace, ref.Name, ref.ID, err))
-						}
-						continue
-					}
-
-				} else {
-					tag = ref.Tag
-					if len(tag) == 0 {
-						tag = "latest"
-					}
-					ist, err2 := c.osClient.ImageStreamTags(ref.Namespace).Get(ref.Name, tag)
-					if err2 != nil {
-						utilruntime.HandleError(fmt.Errorf("failed to get image stream tag %s/%s:%s: %v", ref.Namespace, ref.Name, tag, err2))
-						continue
-					}
-					img = &ist.Image
-					c.imageCache[ref.ID] = img
+			imageID := ref.ID
+			if len(imageID) == 0 {
+				tag = ref.Tag
+				if len(tag) == 0 {
+					tag = "latest"
```

Comment:
Hmm, last time I was looking for it without success. It's actually in `image/api/types.go`.